src/erlcloud_sns.erl

```diff
@@ -240,14 +242,14 @@ get_endpoint_attributes(EndpointArn, Config) ->
             Doc),
     [{arn, EndpointArn} | Decoded].
 
--spec get_endpoint_attributes/3 :: (string(), string(), string()) -> sns_endpoint().
+-spec get_endpoint_attributes/3 :: (parameter(), parameter(), parameter()) -> sns_endpoint().
 get_endpoint_attributes(EndpointArn, AccessKeyID, SecretAccessKey) ->
     get_endpoint_attributes(EndpointArn, new_config(AccessKeyID, SecretAccessKey)).
```

Comment:
Line 247 is too long:     get_endpoint_attributes(EndpointArn, new_config(AccessKeyID, SecretAccessKey))..